.github/autobuild/windows.ps1

```diff
@@ -0,0 +1,169 @@
+{
+    throw "Environment variable jamulus_buildversionstring has to be set to a valid version string"
+}
+
+Function Install-Qt
+{
+    param(
+        [string] $QtVersion,
+        [string] $QtArch
+    )
+    $Args = (
+        "--outputdir", "$QtDir",
+        "windows",
+        "desktop",
+        "$QtVersion",
+        "$QtArch",
+        "--archives", "qtbase", "qttools", "qttranslations", "qtwinextras"
+    )
+    aqt install-qt @Args
+    if (!$?)
+    {
+        echo "WARNING: Qt installation via first aqt run failed, re-starting with different base URL."
+        aqt install-qt -b https://mirrors.ocf.berkeley.edu/qt/ @Args
+        if (!$?)
+        {
+            throw "Qt installation with args @Args failed with exit code $LastExitCode"
+        }
+    }
+}
+
+Function Ensure-Qt
+{
+    if (Test-Path -Path $QtDir)
+    {
+        echo "Using Qt installation from previous run (actions/cache)"
+        return
+    }
+
+    echo "Install Qt..."
+    # Install Qt
+    pip install "aqtinstall==$AqtinstallVersion"
+    if (!$?)
+    {
+        throw "pip install aqtinstall failed with exit code $LastExitCode"
+    }
+
+    echo "Get Qt 64 bit..."
+    Install-Qt "${Qt64Version}" "${Msvc64Version}"
+
+    echo "Get Qt 32 bit..."
+    Install-Qt "${Qt32Version}" "${Msvc32Version}"
+}
+
+Function Ensure-jom
+{
+    choco install --no-progress -y jom --version "${JomVersion}"
+}
+
+Function Ensure-JACK
+{
+    if ($BuildOption -ne "jackonwindows")
+    {
+        return
+    }
+
+    echo "Install JACK2 64-bit..."
+    # Install JACK2 64-bit
+    choco install --no-progress -y jack --version "${JackVersion}"
+    if (!$?)
+    {
+        throw "64bit jack installation failed with exit code $LastExitCode"
+    }
+
+    echo "Install JACK2 32-bit..."
+    # Install JACK2 32-bit (need to force choco install as it detects 64 bits as installed)
+    choco install --no-progress -y -f --forcex86 jack --version "${JackVersion}"
+    if (!$?)
+    {
+        throw "32bit jack installation failed with exit code $LastExitCode"
+    }
+}
+
+Function Build
+{
+    echo "Build installer..."
+    $ExtraArgs = @()
+    if ($BuildOption -ne "")
+    {
+        $ExtraArgs += ("-BuildOption", $BuildOption)
+    }
+    powershell ".\windows\deploy_windows.ps1" "C:\Qt\5.15.2" "C:\Qt\5.15.2" @ExtraArgs
+    if (!$?)
+    {
+        throw "deploy_windows.ps1 failed with exit code $LastExitCode"
+    }
+}
+
+Function Pass-Artifact-to-Job
+{
+    # Add $BuildOption as artifact file name suffix. Shorten "jackonwindows" to just "jack":
+    $ArtifactSuffix = switch -Regex ($BuildOption)
+    {
+        "jackonwindows" { "_jack"; break }
+        "^\S+$"         { "_${BuildOption}"; break }
+        default         { "" }
+    }
+
+    $artifact_deploy_filename = "jamulus_${JamulusVersion}_win${ArtifactSuffix}.exe"
+
+    echo "Copying artifact to ${artifact_deploy_filename}"
+    cp ".\deploy\Jamulus*installer-win.exe" ".\deploy\${artifact_deploy_filename}"
```

Comment:
Maybe the deploy script could be adapted to create the correct name (but that's for another PR)